x-pack/test_serverless/functional/test_suites/common/management/transforms/search_bar_features.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ({ getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['svlCommonPage', 'svlCommonNavigation']);
+
+  const allLabels = [{ search: 'transform', label: 'Data / Transforms', expected: true }];
+  const expectedLabels = allLabels.filter((l) => l.expected);
+  const notExpectedLabels = allLabels.filter((l) => !l.expected);
+
+  describe('Search bar features', () => {
+    before(async () => {
+      await PageObjects.svlCommonPage.login();
+    });
+
+    after(async () => {
+      await PageObjects.svlCommonPage.forceLogout();
+    });
+
+    describe('list features', () => {
+      if (expectedLabels.length > 0) {
+        it('has the correct features enabled', async () => {
+          await PageObjects.svlCommonNavigation.search.showSearch();
+
+          for (const expectedLabel of expectedLabels) {
+            await PageObjects.svlCommonNavigation.search.searchFor(expectedLabel.search);
+            const [result] = await PageObjects.svlCommonNavigation.search.getDisplayedResults();
+            const label = result?.label;
+            expect(label).to.eql(
+              expectedLabel.label,
+              `First result should be ${expectedLabel.label} (got matching items '${label}')`
+            );
+          }
+
+          // await PageObjects.svlCommonNavigation.search.hideSearch();
```

Comment:
Was this meant to be included? If not, can it be removed?

Reply:
Thanks for spotting that, missed reverting the comment after some testing. Fixed in 654f788.